javascript/test/__snapshots__/codeGen.test.ts.snap

```diff
@@ -8,13 +8,13 @@ return function(fury, external) {
   const bw = fury.binaryWriter;
   const cr = fury.classResolver;
   const rr = fury.referenceResolver;
-  const tag_ser_3 = fury.classResolver.getSerializerByTag("example.foo.1");
-  const tag_ser_4 = fury.classResolver.getSerializerByTag("example.foo.2");
-  const tag_ser_12 = fury.classResolver.getSerializerByTag("example.bar.1");
-  const tag_ser_13 = fury.classResolver.getSerializerByTag("example.bar.2");
-  const tagWriter_19 = cr.createTagWriter("tuple-object-wrapper");
+  const tag_ser_4 = fury.classResolver.getSerializerByTag("example.foo.1");
```

Comment:
Could we avoid to commit `ts.snap` to repo?

Reply:
.snap is used in unittest, it can ensure the generated code to match our expectation

Reply:
Also used for reviewing the generated code on GitHub and tracing the code changes with Git.

Reply:
Also used for reviewing the generated code on GitHub and tracing the code changes with Git.

Reply:
Can we use unittest to ensure the correctness of generated code. If we have a complicated class, we may generate thounds of lines of code. In fury java, I have see some big objects generated 6000 line of code.